experiments/long_runs/leaderboard/data_sources.jl

```diff
@@ -0,0 +1,142 @@
+import ClimaAnalysis
```

Comment:
I noticed that we change the units of the simulation output in some cases (GPP), and in the observations (LWU) in others. it would be good to make this consistent (simulation unit changes to match observations)

Reply:
I agree that this should be consistent, but I think we should change all observation units to match simulation units. This is because there is some consistency with how the units are written for the simulation data, but not for the observational data.

Reply:
For some variables, it is more natural to use observation units instead of simulation units, so we should convert towards what makes the most sense

Reply:
fair enough!